diff --git a/src/dmmf/utils.ts b/src/dmmf/utils.ts
index 677e4764a44cad37c23b4564cf758857f133406f..1abfe2f7deea92a4a63af39abf2255f5f8197da8 100644
--- a/src/dmmf/utils.ts
+++ b/src/dmmf/utils.ts
@@ -3,16 +3,18 @@ import { DMMF } from '@prisma/client/runtime'
 export const getPrismaClientDmmf = (packagePath: string): DMMF.Document => {
   let dmmf: undefined | DMMF.Document = undefined
 
-  try {
+  console.log("in getPrismaClientDmmf with package path", packagePath)
+  //try {
     const prismaPackage = require(packagePath)
     dmmf = prismaPackage.dmmf || prismaPackage.Prisma.dmmf
-  } catch (error) {
+  /*} catch (error) {
     throw new Error(
       `Failed to import prisma client package at ${packagePath}. The following error occurred while trying:
         ${(error as Error).stack}`
     )
-  }
+  }*/
 
+  console.log("final dmmf", dmmf)
   if (!dmmf) {
     throw new Error(`\
 You most likely forgot to initialize the Prisma Client. Please run \`prisma generate\` and try to run it again.