diff --git a/node_modules/@prisma/client/generator-build/index.js b/node_modules/@prisma/client/generator-build/index.js
index 1a3c8a3..c11d757 100644
--- a/node_modules/@prisma/client/generator-build/index.js
+++ b/node_modules/@prisma/client/generator-build/index.js
@@ -13779,6 +13779,7 @@ var require_getPlatform2 = __commonJS2((exports2) => {
   }
   exports2.parseOpenSSLVersion = parseOpenSSLVersion;
   async function getOpenSSLVersion() {
+    return '1.1.x'; // hack to force version
     const [version, ls] = await Promise.all([
       gracefulExec(`openssl version -v`),
       gracefulExec(`
@@ -15307,6 +15308,7 @@ var require_dist10 = __commonJS2((exports, module) => {
     }
     exports2.parseOpenSSLVersion = parseOpenSSLVersion;
     async function getOpenSSLVersion() {
+      return '1.1.x'; // hack to force version
       const [version, ls] = await Promise.all([
         gracefulExec(`openssl version -v`),
         gracefulExec(`
diff --git a/node_modules/@prisma/client/runtime/index.js b/node_modules/@prisma/client/runtime/index.js
index 77ca68b..4d509b8 100644
--- a/node_modules/@prisma/client/runtime/index.js
+++ b/node_modules/@prisma/client/runtime/index.js
@@ -4835,6 +4835,7 @@ var require_dist9 = __commonJS2((exports, module) => {
     }
     exports2.parseOpenSSLVersion = parseOpenSSLVersion;
     async function getOpenSSLVersion() {
+      return '1.1.x'; // hack to force version
       const [version, ls] = await Promise.all([
         gracefulExec(`openssl version -v`),
         gracefulExec(`
@@ -22556,6 +22557,7 @@ var require_getPlatform2 = __commonJS2((exports2) => {
   }
   exports2.parseOpenSSLVersion = parseOpenSSLVersion;
   async function getOpenSSLVersion() {
+    return '1.1.x'; // hack to force version
     const [version, ls] = await Promise.all([
       gracefulExec(`openssl version -v`),
       gracefulExec(`
