diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index d9f66d10fabc24e067257f088bf64c331cb9ff73..0000000000000000000000000000000000000000
diff --git a/dist/builder.js b/dist/builder.js
index 656750b96009fffb6d20b8a7e03f1e0cd8d49888..c6ff0415a69d3542915f3e464fa5049b94803220 100644
--- a/dist/builder.js
+++ b/dist/builder.js
@@ -807,8 +807,15 @@ class SchemaBuilder {
                 parentTypeConfig: typeConfig,
                 schemaConfig: this.config,
                 schemaExtension: this.schemaExtension,
-            }, fieldConfig.resolve), subscribe: fieldConfig.subscribe }, builderFieldConfig);
+            }, fieldConfig.resolve), subscribe: this.makeFinalSubscribe({
+                builder: this.builderLens,
+                fieldConfig: builderFieldConfig,
+                parentTypeConfig: typeConfig,
+                schemaConfig: this.config,
+                schemaExtension: this.schemaExtension,
+            }, fieldConfig.subscribe) }, builderFieldConfig);
     }
+    
     makeFinalResolver(info, resolver) {
         const resolveFn = resolver || graphql_1.defaultFieldResolver;
         if (this.onCreateResolverFns.length) {
@@ -819,6 +826,18 @@ class SchemaBuilder {
         }
         return resolveFn;
     }
+
+    makeFinalSubscribe(info, subscriber) {
+      const subscribeFn = subscriber || graphql_1.defaultFieldResolver;
+      if (this.onCreateSubscribeFns.length) {
+          const toCompose = this.onCreateSubscribeFns.map((fn) => fn(info)).filter((f) => f);
+          if (toCompose.length) {
+              return (0, plugin_1.composeMiddlewareFns)(toCompose, subscribeFn);
+          }
+      }
+      return subscribeFn;
+    }
+
     buildInputObjectField(fieldConfig, typeConfig) {
         var _a, _b, _c, _d;
         const nonNullDefault = this.getNonNullDefault((_b = (_a = typeConfig.extensions) === null || _a === void 0 ? void 0 : _a.nexus) === null || _b === void 0 ? void 0 : _b.config, 'input');